aten/src/ATen/native/native_functions.yaml

```diff
@@ -5351,7 +5351,7 @@
 
 - func: _validate_sparse_coo_tensor_args(Tensor indices, Tensor values, int[] size) -> ()
 
-- func: _validate_sparse_csr_tensor_args(Tensor crow_indices, Tensor col_indices, Tensor values, int[] size) -> ()
+- func: _validate_sparse_csr_tensor_args(Tensor crow_indices, Tensor col_indices, Tensor values, int[] size, Layout? layout=None) -> ()
```

Comment:
Can you make this a keyword-only argument to avoid BC issues?

Reply:
Resolved via https://github.com/pytorch/pytorch/pull/75946